### PR TITLE
issue-97-typos-in-readme fixed the typos in readme and node path for …

### PR DESCRIPTION
**Pull request by frozan**
https://github.com/pipedrive/client-nodejs/issues/97

## Changes
- Updated the typos in Readme.md
- Added the node path for running the repo on windows

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "url": "http://github.com/pipedrive/client-nodejs.git"
   },
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha ./test/unit"
+	"test": "./node_modules/mocha/bin/mocha ./test/unit",
```
Comment (eerootsus) [inline package.json]: @frozan @RuTsEST 
I remember that platform specifics used to be a problem, but this was a long time ago. It makes no sense to have a separate command for Windows and honestly I have not seen this used anywhere.

I think `node ./node_modules/mocha/bin/mocha ./test/unit` might work (https://shapeshed.com/writing-cross-platform-node/) on both. I don't have Windows to test.

Comment (frozan) [reply package.json]: Yes, you are right about the path. I was using a Windows OS when I pulled this repo and ran into that path error but it worked fine when I ran it on Ubuntu. I thought to specify it for future reference.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This is the official Pipedrive API wrapper-client for NodeJS based apps, distrib
 # Install
 
 ```
-npm install pipedrive
```
Comment (tot-ra) [inline README.md]: This refers to https://www.npmjs.com/package/pipedrive
so its supposed to include package name, you can't just remove it

**Comment by tot-ra** [comment]
talked directly to Frozan, fixed `npm run test` in readme, otherwise makes no sense to change installation, will close PR

**Pull request closed without merging**